js/packages/seed-vault/package.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "@solana-mobile/seed-vault-lib",
+    "description": "A React Native wrapper of the Solana Mobile, Seed Vault SDK. Apps can use this to interact with seed vault implementations on Android",
+    "version": "0.0.1",
+    "author": "Marco Martinez <marco.martinez@solanamobile.com>",
+    "repository": "https://github.com/solana-mobile/sed-vault-sdk",
+    "license": "Apache-2.0",
+    "type": "module",
+    "sideEffects": false,
+    "main": "lib/cjs/index.js",
+    "react-native": "lib/cjs/index.native.js",
+    "module": "lib/esm/index.js",
+    "types": "lib/types/index.d.ts",
+    "browser": {
+        "./lib/cjs/index.js": "./lib/cjs/index.browser.js",
+        "./lib/esm/index.js": "./lib/esm/index.browser.js"
+    },
```

Comment:
Kill all the `browser` and `node` exports. Make this package.json only work with React Native.